### DISCUSSION TITLE
Make the tapping on the message bubble returns more meaning details

### DISCUSSION THREAD

**Discussion by realvee** [flyerhq/flutter_chat_ui#664] [Ideas & Requests]
## Is your feature request related to a problem?
I think it'd be more flexible for us if the tapping on the bubble can return position of each bubble, so we can know which bubble is touched on -- so we can play around that. 

## What solution would you suggest?
Return TapDownDetails and LongPressStartDetails for each event on GestureDetector for the bubble

## Is there any additional solution to that?
Nope

## Extras
**Screenshots or videos 📸**
If applicable, add screenshots or videos to help explain your feature.

**Code snippets 📝**
Instead of using the tap events :
    onDoubleTap: () => onMessageDoubleTap?.call(context, message),
    onLongPress: () => onMessageLongPress?.call(context, message),
    onTap: () => onMessageTap?.call(context, message),

I did replace these to : 
    onDoubleTapDown: (details) => onMessageDoubleTap?.call(context, message, details),
    onLongPressStart: (details) =>  onMessageLongPress?.call(context, message, details),
    onTapDown: (details) => onMessageTap?.call(context, message, details),

We might need to follow to edit some few more lines to correspond to the changes.

**Comment by demchenkoalex** [comment]
thanks for the suggestion! will add in `v2`

**Comment by demchenkoalex** [comment]
not closing because I need first to add to backlog so I don't forget :)
